src/nvector/manyvector/nvector_manyvector.c

```diff
@@ -540,18 +544,15 @@ void MVAPPEND(N_VSpace)(N_Vector v, sunindextype *lrw, sunindextype *liw)
 
 #ifdef MANYVECTOR_BUILD_WITH_MPI
 /* This function retrieves the MPI Communicator from an MPIManyVector object. */
-void *N_VGetCommunicator_MPIManyVector(N_Vector v)
+SUNComm N_VGetCommunicator_MPIManyVector(N_Vector v)
```

Comment:
If we choose to use `MPI_Comm` for MPI-specific modules, then this should be updated to `MPI_Comm`.